raiden/messages.py

```diff
@@ -399,7 +430,7 @@ class Ping(SignedMessage):
 
 @dataclass(repr=False, eq=False)
 class SecretRequest(SignedRetrieableMessage):
-    """ Requests the secret which unlocks a secrethash. """
+    """ Reveals the secret/preimage which unlocks a lock. """
```

Comment:
> Reveals the secret

Doesn't it simply request the secret to be revealed (through `secrethash`), instead of **_revealing_** the secret, which is done by `RevealSecret`?